creator/data_templates/mutations/data_template.py

```diff
@@ -0,0 +1,217 @@
+import graphene
+from graphql import GraphQLError
+from graphql_relay import from_global_id
+from django.conf import settings
+
+from creator.organizations.models import Organization
+from creator.data_templates.models import DataTemplate
+from creator.data_templates.nodes.data_template import DataTemplateNode
+
+
+class CreateDataTemplateInput(graphene.InputObjectType):
```

Comment:
for consistency with past mutation we might not want to change this but I've found it is nice to group mutation by data type rather than action. So dataTemplateCreate, dataTemplateUpdate. It helps keeps all the dataTemplateActions together in the docs.

This idea comes from https://github.com/Shopify/graphql-design-tutorial/blob/master/TUTORIAL.md#input-structure-part-1